### ISSUE TITLE
Question about the Frequency on v1/browse/new-releases

### ISSUE THREAD

**Issue by Saik19922** [spotify/web-api#1390]
I'm building a mobile/web/desktop app that intends to notify the user if a band he follows uploads a release. For this I'm currently hitting the new-releases endpoint once an hour since 3 days but I haven't had a single change in the response it returns since then.

So my two questions are:

How often-updated and how trustable (as in has "all" new releases that do hit spotify) is that endpoint.

If it's neither trustable enough to mirror all releases or not frequently updated enough: What would the proper way of hitting the spotify API for new releases be?

**Comment by dfconroy** [comment]
The new releases endpoint is pretty much a TOP 50ish listing. If users are following some not-so-known artists it's likely that their new albums won't be returned in this API call.

I wrote an app that does the same thing you're trying and this is how I accomplish the feat.

1. Grab all my following artists.

2. Loop through each artists and grab all their albums.
NOTE. Have to grab all of them until Spotify provides a filter field for the release date.

https://api.spotify.com/v1/artists/{id}/albums

3. Loop through those albums and only add albums added within last 7 days to an array.

album.ReleaseDate

4. Take all albums added within last 7 days and loop through all of them to get the tracks. 
NOTE. I use the several albums API so I can do this in a loop of 20 albums at a time.

https://api.spotify.com/v1/albums


5. Loop throguh all the tracks and add the tracks to a playlist.

**Comment by Saik19922** [comment]
So, a bit of background information from me here.

Users won't be limited in which artist they follow. This means If - potentially - I'm reaching 1000 users that each follow 10 different artists (fictionally saying, none of them follow the same artist) I'd have to loop through 10.000 artists, grab all their albums (I'm focussing on albums/single's as those will naturally come with songs, so I'm not dipping into albums on my backend), deem if it's a new release by checking the release date and then (since I'm doing notifications) deem if I already know this release.

Understood so far, should be doable (albeit slightly annoying that spotify doesn't list actual new releases and only lists a "featured list").

Some follow ups for you since you apparently know that topic.

Is there any specific time spotify tends to have releases uploaded? If so, which?
Whats the rate limit for server-to-server communication? (Most likely will have to deal with throttling).

**Comment by dfconroy** [comment]
From what I noticed new releases can be added daily, but I noticed the most on Wednesday through Fridays. 

Here are my counts of new releases I've found on those days:

Wednesday	596
Thursday	3560
Friday	3368

And from my testing, I can get about 10-20 requests per second. Though, spotify provides the nifty 'retry-after' field on it's response that lets you know how long to wait till you try again. 

https://stackoverflow.com/questions/30548073/spotify-web-api-rate-limits
